WordPress/gutenberg
# Pullquote block: avoid text-align settings affecting block width and font size

## What?

Pullquote block: avoid text-align settings affecting block width and font size

## Why?

Because text-align settings should not affect the width of the block or the font size.

## How?

We are removing CSS that adds width limits or font sizes depending on the text-align of the block.

## Testing Instructions
1. Select emptytheme theme to discard theme styles problems.
2. Add a new post
3. Add a group block
4. Inside the group block add a pullquote block
5. Select the align left or right.
6. Save and publish
7. See the frontend and verify that the width of the block is wrongly limited by CSS  and the font size is not the same using different text-align option.


Test markup:
```

<div class="wp-block-group">
<figure class="wp-block-pullquote has-text-align-left"><blockquote><p>This is my quote</p><cite>The author</cite></blockquote></figure>
</div>

```

## Screenshots or screencast 
**Before:** 

https://user-images.githubusercontent.com/1310626/184364870-25516cae-5487-49e2-8df4-63be98ebcf12.mp4

**After:**

https://user-images.githubusercontent.com/1310626/184367632-46975ae9-790a-4c80-bd81-7691a5b77399.mp4


Fixes: https://github.com/WordPress/gutenberg/issues/43187
